hooks/collector.py

```diff
@@ -119,6 +119,8 @@ def _add_item(self, settings, parent_item, item_name, item_type, context=None, p
         type_display = item_info["type_display"]
         icon_path    = item_info["icon_path"]
 
+        self.logger.info("Added %s of type %s" % (item_name, item_type))
```

Comment:
Feels unnecessary for the end user, self.logger.debug?

Reply:
Ah yes, this was more of a debug statement.